docs/source/i18n/ultimate.hi.rst

```diff
@@ -0,0 +1,26 @@
+.. _chap_consent_ultimate_hi:
+
+Hindi
+~~~~~~~
+(translation courtesy of Hardik Kothare and Saurabh Chavan)
+
+Single access type version (all data shared publicly; recommended)
+------------------------------------------------------------------
+
+**Version:** OBC-ULT.hi 1.0.0
+
+वर्तमान अभ्यास में प्राप्त किए हुए डेटा का प्रयोग इस प्रकल्प के अलावा भविष्य में अन्य अनुसंधान प्रकल्पों में भी किया जा सकता है। भविष्य के उन प्रकल्पों का ध्येय इस प्रकल्प से भिन्न हो सकता है। हम यह संग्रहित डेटा, विकिरण प्रतिमाओं समेत, आम जनता को इंटर्नेट और अप्रतिबंधित डेटाबेस द्वारा उपलब्ध करवाएँगे।
```

Comment:
प्रकल्प  means project (v) as in projection, not project (n), better word seems to be परियोजना.
Same with अभ्यास, it translates to study as in teaching/learning related, better word is अध्ययन to mean study something as in research.

Google translate's version for first sentence is actually pretty good to use (replacing अभ्यास with अध्ययन) :
"इस परियोजना के अलावा, वर्तमान अभ्यास में प्राप्त डेटा का उपयोग भविष्य में अन्य अनुसंधान परियोजनाओं में किया जा सकता है।

Reply:
Although परियोजना is a suitable replacement, प्रकल्प is still a noun and not a verb (see: https://www.shabdkosh.com/search-dictionary?lc=hi&sl=en&tl=hi&e=prakalp).
Please cite a source for "प्रकल्प means project (v) as in projection, not project (n)". 

Agree on replacing अभ्यास with अध्ययन and प्रकल्प with परियोजना.

Reply:
> Based on my comment, this is definitely not ready to merge.

Please read through the rest of the translation to suggest more changes.

Reply:
Sorry, I should have been clear in my comment. I meant the meaning of it is closer to "project(v)" in English as in "projection", not "Project (n)" as in "Project Management".
I can't find a citation online at the moment, but from my knowledge of language, that is how I understand it.

Other issues with this sentence are:
- Can you explain this phrase "डेटा का प्रयोग"?
- Should "यह" be "यहा"?

I will read the whole thing and comment later, thanks.

Reply:
English "project" as in to forecast or to predict is अनुमान करना, while "projecting" a film etc. on a surface is प्रक्षेपित करना. "Project Management" or anything related to the meaning of a project as an undertaking (research or otherwise) is in fact the _principle_ meaning conveyed by प्रकल्प. The word प्रकल्प is regularly used to talk about schemes and projects. A quick internet search of government or NGO websites in Hindi supports this.
Just one example here from a leading Hindi newspaper. 
https://www.bhaskar.com/local/rajasthan/nagaur/makrana/news/ppe-kits-presented-in-hospital-127356424.html

Also attaching a screenshot from the Hindi lexicon by IIT Bombay. 
प्रकल्प is a noun and it means a "project" as in something done together by people, a perfect synonym of परियोजना.
<img width="1438" alt="Screen Shot 2020-05-31 at 2 57 48 PM" src="https://user-images.githubusercontent.com/44656627/83363585-69250880-a34f-11ea-824f-311e6624ef40.png">

"डेटा का प्रयोग" and "डेटा का उपयोग" mean the same - "the use of this/these data" - "डेटा का इस्तेमाल" is an equivalent expression - however the latter is more Urdu than standard Hindi.
I can cite sources to support the meaning of प्रयोग as "use" if needed. Also, if you reverse translate "प्रयोग" - it shows up as "use".

Should "यह" be "यहा"? --> यहा is not a Hindi word at all.
यह = this. 
The Indic group of Indo-European languages do not have the indefinite / definite articles a/the. Instead, in Hindi, we would use यह/ये and वह/वे to replace the "the" in "the data we are collecting". In short, Hindi uses demonstrative adjectives in lieu of the definite/indefinite articles.
Additionally, the word you might be thinking of is यहाँ which means "over here" and not "the/this/these". "Over here" is not the intended meaning of "हम यह संग्रहित डेटा..." as the equivalent of "the data we are collecting".
The reason we did not rely on direct google translations from the get go is that google translations will ignore context and nuanced idiomatic expressions or specialised semantics.

Reply:
Thanks for providing the references. Given your example usages, it sound good to me.
yaha vs ye is typo on my part and the rest of my comments are based on my colloquial knowledge of Hindi and I am not proficient in language in official documents. 🤷‍♂️

Reply:
No worries - but on digging a little deeper - we found that अभ्यास and अध्ययन are synonyms too. In both the senses -- 1) formal education / teaching / learning related and 2) a deliberate attempt to understand a specialised topic i.e. a research objective.
अध्याय = a chapter (of a book etc. that one would teach / learn the contents therein.)
I am attaching screenshots for both the words. 
<img width="1360" alt="Screen Shot 2020-05-31 at 4 01 59 PM" src="https://user-images.githubusercontent.com/44656627/83364716-1c91fb00-a358-11ea-944e-da3844a67ef5.png">
<img width="1359" alt="Screen Shot 2020-05-31 at 4 03 22 PM" src="https://user-images.githubusercontent.com/44656627/83364732-49461280-a358-11ea-917e-321c4a49ed92.png">
This is not the sole resource I am using and I have multiple sources that I am not providing here to avoid cluttering. Please let us know if it is desirable to have a look at these other sources and I will post them here for everyone's benefit.
Thank you.